### PR TITLE
Stats: Update to a higher resolution Woo icon.

### LINKED ISSUE #70462: Component: Add higher resolution Woo mobile app icon
The `MobilePromoCard` component supports displaying the Woo mobile app at 55x56 points. The included PNG file is only 55x56 pixels though, meaning the image is a bit fuzzy on HiDPI displays.

We should update the component with a higher resolution (at least 110x112 pixels) to better support Retina/HiDPI displays.

### PR DESCRIPTION
**Pull request by a8ck3n**
#### Proposed Changes

Adds a higher resolution Woo icon and controls the display size via CSS. This fixes the fuzzy icon on HiDPI screens.

#### Testing Instructions

* Check out this branch.
* Run `yarn workspace @automattic/components run storybook` to fire up Storybook.
* View the Mobile Promo Card → Default to see the Jetpack card.
* Toggle to the Woo card and confirm the layout doesn't shift around.
* Confirm the Woo icon doesn't look fuzzy on a HiDPI/Retina display.

#### Pre-merge Checklist

- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

Related to #70462.

### CODE REVIEW

**Review by wilmersondasilva — APPROVED** [review]
LGTM!

File: packages/components/src/mobile-promo-card/style.scss [hunk 1]
```diff
@@ -51,6 +51,12 @@
 
 .promo-card__icons {
 	margin-bottom: 24px;
+	max-height: 55px;
+
+	.woo-icon {
+		height: 55px;
```
Comment (grzegorz-cp) [inline packages/components/src/mobile-promo-card/style.scss:57]: Since those two values are linked, you might want to use a variable.

**Review by grzegorz-cp — APPROVED** [review]
Looks good, but I left one suggestion. It's good to merge without it if you don't think it's necessary.

**Review by jsnmoon — APPROVED** [review]
Just piling on 😁

File: packages/components/src/mobile-promo-card/style.scss (hunk 1 above)
Comment (a8ck3n) [reply packages/components/src/mobile-promo-card/style.scss:57]: Oh, good call! I'll definitely add that.

**Pull request merged**